fleetctl/fleetctl.go

```diff
@@ -745,6 +782,62 @@ func setTargetStateOfUnits(units []string, state job.JobState) ([]*schema.Unit,
 	return triggered, nil
 }
 
+// getArgMaxAttempts gets the correct value of how many attempts to try
```

Comment:
yes getBlockAttempts() but no arguments, I didn't want to make sharedFlags a type, so it will just access its fields as it is done inside other part of the code inside fleetctl.go. It's better this way.